src/transport/raw/BLE.h

```diff
@@ -91,6 +122,20 @@ class DLL_EXPORT BLEBase : public Base, public Ble::BleLayerDelegate
 
     CHIP_ERROR SetEndPoint(Ble::BLEEndPoint * endPoint) override;
 
+    /**
+     * Change BLE transport to this
+     *
+     * This is relevant when there is more than one TransportMgr,
+     * for example, when a device is both a commissioner (has a CHIPDeviceController)
+     * and a commissionee (has a Server) since each TransportMgr will set
+     * the global BleLayer transport to point to itself.
+     *
+     * In this scenario, the device will need the ability to toggle between a
+     * BleLayer transport for commissioner functionality and one for commissionee functionality.
+     */
+    void SetBleLayerTransportToSelf() { mBleLayer->mBleTransport = this; }
```

Comment:
This method is never called, nor is the method below.

Reply:
Added calls from CHIPDeviceController when commissioning over BLE